demisto_sdk/commands/content_graph/common.py

```diff
@@ -145,7 +150,7 @@ def metadata_name(self) -> str:
             return "automation"
         elif self == ContentType.INDICATOR_TYPE:
             return "reputation"
-        elif self == ContentType.LAYOUT:
+        elif self == ContentType.LAYOUT or self == ContentType.CASE_LAYOUT:
```

Comment:
You could turn this into a constant group, or just do like this
```suggestion
        elif self in (ContentType.LAYOUT, ContentType.CASE_LAYOUT):
```